LondonBritishLibrary/orient/BLorient13274.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="BLorient13274">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Magic Scroll</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 13274</idno>
+                        <altIdentifier><idno>Or. 13274</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 74</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title ref="LIT1758Lefafa" type="incomplete"/>
+                            <msItem xml:id="ms_i1.1">
+                                <title ref="LIT1758Lefafa#FirstChapter"/>
+                                <incipit xml:lang="gez">በስመ፡<gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ መድኃኒተ፡ ነፍስ፡ ወሥጋ፡ መጽሐፈ፡ ሕይወት፡ ወመጽሐፈ፡ መድኃኒት፡
+                                    ልፋፈ፡ ጽድቅ፡ ዘጸሐፈ፡ አብ፡ በእዲሁ፡ እምቅድመ፡ ይትፈጠሩ፡ ሰማይ፡ ወምድር፡ ወእምቅድመ፡ ይትወለድ፡ ክርስቶስ፡ እምእግዝእትነ፡ ቅድስት፡ ድንግል፡ በ፪፡
```

Comment:
maybe add a "sic", as this is not the expected form (although I see that this is what Strelcyn has)?

Reply:
I think, you speak about በእዲሁ፡?

Reply:
Yes, sorry for not mentioning what I was talking about :)